Extensions/GoogleOptimize/Src/Tasks/GoogleOptimize/models/TaskParameter.ts

```diff
@@ -0,0 +1,119 @@
+import * as tl from 'azure-pipelines-task-lib/task';
+
+export class TaskParameter {
+	private _endpoint: string;
+	private _accountId: string;
+	private _webPropertyId: string;
+	private _profileId: string;
+	private _experimentId: string;
+	private _action: string;
+	private _trafficCoverage: string | null;
+	private _equalWeighting: string | null;
+	private _filePath: string | null;
+
+	private static _taskParameters: TaskParameter = null;
+
+	constructor () {
```

Comment:
Make constructor private so a new instance cannot be created from outside the class and getInstance() is the only way to get an instance of this class.